ui/app/templates/components/auth-config-form/options.hbs

```diff
@@ -12,13 +12,12 @@
     {{/each}}
   </div>
   <div class="field is-grouped box is-fullwidth is-bottomless">
-    <button
+    <Hds::Button
+      @text="Update Options"
```

Comment:
Noticed we usually lowercase the second word in button text. This is the text from before, but do we want to clean this up as we come across it?